cupy/cuda/memory.pxd

```diff
@@ -7,23 +7,29 @@ from cupy.cuda cimport device
 
 
 @cython.no_gc
-cdef class Memory:
+cdef class BaseMemory:
 
     cdef:
         public size_t ptr
         public Py_ssize_t size
         public int device_id
 
 
+cdef class UnownedMemory(BaseMemory):
+
+    cdef:
+        public object owner
```

Comment:
Why do you use `public`?

Reply:
I made it public because of this TODO:
https://github.com/cupy/cupy/pull/1610/commits/487b4dad1f053eb9d9e2e43ec2b54d2c0749a7c1#diff-4ddbad27311cb0c1a9a353025e48ded1R17

`cupy.ndarray` should be able to retrieve it.

Reply:
As it's still TODO, it's OK to make it private.

Reply:
Fixed.